ext/com_dotnet/com_persist.c

```diff
@@ -276,8 +276,7 @@ PHP_COM_DOTNET_API IStream *php_com_wrapper_export_stream(php_stream *stream)
 	return (IStream*)stm;
 }
 
-#define CPH_ME(fname, arginfo)	PHP_ME(com_persist, fname, arginfo, ZEND_ACC_PUBLIC)
-#define CPH_METHOD(fname)		static PHP_METHOD(com_persist, fname)
+#define CPH_METHOD(fname)		static PHP_METHOD(COMPersistHelper, fname)
```

Comment:
```suggestion
#define CPH_METHOD(fname)		PHP_METHOD(COMPersistHelper, fname)
```
Not necessary? I think we had to do this for other extensions, but possibly Windows allows a static definition with a non-static declaration just fine?

Reply:
Oh, yes, I missed to spot this. I had to remove a bunch of `static` modifiers for sure.